staging/nvidia/charts/grafana/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Just to make sure... these are Apache licensed because they're copied from upstream?

Reply:
no, they are not copied from anywhere. We created this sub-chart for grafana dashboard. Added this license because other Nvidia charts are under apache 2.0. Should we remove it?

Reply:
Let's keep the liscense because we developed this dashboard based on top of some contents in this repo https://github.com/NVIDIA/gpu-monitoring-tools